### PR TITLE
✨ Don't count intro against user if solo

### LINKED ISSUE #45: Only play entrance sound when more than one person is in the channel
Bot will play the entrance sound when I join a channel and it's just me, should only play entrance sound when there is at least one other person in the channel

### PR DESCRIPTION
**Pull request by dudeofawesome**
Also fixes song playback. We had a time unit error.

Addresses #45's additional comment.

### CODE REVIEW

File: src/bot/events/intro-music.ts [hunk 1]
```diff
@@ -67,18 +65,31 @@ export class IntroMusic {
...
+          differenceInMilliseconds(
+            now,
+            virgin.intro_last_played ?? new Date(0),
+          ),
         ) >=
         (virgin.intro_song?.computed_timeout_ms ??
           configuration.audio.default_intro.timeout_ms)
       ) {
+        // channel only has 1 member
+        if ((new_state.channel?.members.size ?? 0) >= 2) {
```
Comment (EdgarSaldivar) [inline src/bot/events/intro-music.ts:77]: The bot joins after this line so shouldnt this be '>=1' ? or am i misunderstanding here?

Comment (dudeofawesome) [reply src/bot/events/intro-music.ts:77]: We're checking to see if we should count this play against the user's timeout, so we want to see if the user was in there on their own or not.

Comment (EdgarSaldivar) [reply src/bot/events/intro-music.ts:77]: Yes but why 2 and not 1? If they are in there by themselves should the members size be 1?

Comment (dudeofawesome) [reply src/bot/events/intro-music.ts:77]: We only want to set their `intro_last_played` if anyone else heard it, as in, there are 2 or more people in the chat.

Comment (EdgarSaldivar) [reply src/bot/events/intro-music.ts:77]: ahhh my bad im dumb i misunderstood.

**Pull request merged**